apps/frontend/src/layouts/AppLayout.tsx

```diff
@@ -45,7 +45,11 @@ export default function AppLayout({
   return (
     <OnlyMobileLayout>
       {back && (
-        <div className="sticky top-0 h-12 flex items-center border-b border-b-quaternary/20 bg-white z-20">
+        <div className={cn(
+          "sticky top-0 h-12 flex items-center bg-white z-20",
+          headerDivider && "border-b border-b-quaternary/20"
+        )}
+        >
```

Comment:
This allows the divider to be removed from the top header (the header with the back button).